lib/Service/FeedServiceV2.php

```diff
@@ -257,7 +257,7 @@ public function fetch(Entity $feed): Entity
              */
             list($fetchedFeed, $items) = $this->feedFetcher->fetch(
                 $location,
-                false,
+                true,
```

Comment:
If we want to fetch it every run I think we should just abandon the option. We're saving kilobytes at most anyway.

Reply:
yea I think it makes sense as users might have feeds for years and the website might change the logo adjust it a little bit and so on

Reply:
I agree, but I think we shouldn't have `if ($favicon === true)` anymore then and just always update the favicon. That also makes this function signature easier.

Reply:
Refetching would break my current setup.
I've replaced every favicon with a base64 encoded `data:` uri of the icon to avoid pinging every server each time I open the web interface (e.g. YouTube: `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAACXBIWXMAAAsSAAALEgHS3X78AAABDklEQVRYw+2X4Y2CQBSEv3e5ArgOKIEO3BK0A0uhBDsRO8AOtgMpYTuY+yErRL3cobLrJUwyISGEmWTfm/fWJJETH2TGYiC7gc+7b80KoALKngBu4r/b/tn19Ejh5itJA6EStALNxFZQjTWvxcOM4pFhbGJsoEkgHtlEXbsEkVnaRJJs6AKzMnn5nwv90obTDKzXr7BQPZ4D+z20LZRlxiBarcB7qOtna0EI3KQqvob3knNTO8ENbfisgYjdTiqKSQayz4J/fgQhSHX9aBo6ST9Mw7/geITtFrpuhnH8GzYbaJrXBKKkGMWnxOX3hRTeZBidcUgof7jdiLIvJJlWMrt7M0q4lNpyNVsM5DbwDTlqsJtbbh7VAAAAAElFTkSuQmCC`).
However, I don't want to stall an improvement for everyone with my specific case. 😉

Reply:
Yea maybe in the future we can add a general private mode but I think for now it's out of scope, it's in the nature of our app to leak some data, the android app introduced a private mode but in the first design meeting we came to the conclusion that it needs improvement.

Maybe the new UI can learn from that, if we ever get it else with NC22 we become an api only app and don't have to care anymore hehe

Reply:
https://davidwalsh.name/data-uri-php this looks easy enough to add (in a separate PR) and it might speed up load times as well. But we definitely can't take it into consideration until it's a real feature.

Reply:
Yes, converting to a data-URI is simple .. but I've had multiple clients which didn't support it (but thats the topic of another PR as you said)